rainforest/reporter.go

```diff
@@ -26,14 +26,22 @@ type RunStepDetails struct {
 	Browsers []RunBrowserDetails `json:"browsers"`
 }
 
+// RunTestBrowserDetails contain browsers for each run test
+type RunTestBrowserDetails struct {
+	Name   string `json:"name"`
+	Result string `json:"result"`
+}
+
 // RunTestDetails contains details about a Run Test
 type RunTestDetails struct {
-	ID        int              `json:"id"`
-	Title     string           `json:"title"`
-	CreatedAt time.Time        `json:"created_at"`
-	UpdatedAt time.Time        `json:"updated_at"`
-	Result    string           `json:"result"`
-	Steps     []RunStepDetails `json:"steps"`
+	ID             int                     `json:"id"`
+	Title          string                  `json:"title"`
+	CreatedAt      time.Time               `json:"created_at"`
+	UpdatedAt      time.Time               `json:"updated_at"`
+	Result         string                  `json:"result"`
+	Steps          []RunStepDetails        `json:"steps"`
+	HasWispResults bool                    `json:"has_wisp_results"`
```

Comment:
Nitpick - is `Wisp` here something we could change to say e.g. `HasRfaResults`, or did that ship already sail?

Reply:
We can change the field name, we probably can change the API as well since it's not public

Reply:
Done!